client/src/components/CatalogTourCard/CatalogTourCard.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Stack, CardActions, CardMedia, Button, Typography, CardContent } from '@mui/material';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import { ReactComponent as CoinsIcon } from '../../svg/CoinsIcon.svg';
+import styles from './CatalogTourCard.module.scss';
+
+function CatalogTourCard() {
+  return (
+    <Stack
+      direction={{ xs: 'column', tablet: 'row' }}
+      sx={{ backgroundColor: 'white', borderRadius: '10px', boxShadow: '0px 0px 40px rgba(0, 0, 0, 0.05)' }}
+      spacing={1}
+    >
+      <CardMedia
+        sx={{
+          width: {
+            xs: '100%',
+            tablet: '238px',
+          },
+          height: {
+            xs: '240px',
+            tablet: 'auto',
+          },
+          borderRadius: {
+            xs: '10px 10px 0 0',
+            tablet: '10px 0 0 10px',
+          },
+        }}
+        component='img'
+        image='https://visitukraine.today/media/tours/gallery/ALeR7GgYjAqCqfJnQX5ZYKnBcsDZ6MTJs77IIBKi.jpg'
+        alt='tour photo'
+      />
+      <CardContent sx={{ padding: '0 30px 0 36px' }}>
+        <Typography
+          sx={{ color: 'black', marginBottom: '20px', marginTop: '20px' }}
+          variant='h3'
+          component='div'
```

Comment:
I think it is better to leave this component to be `<h3>`, or change to `<p>`. To my mind, `<div>` is not semantically correct tag to use with text

Reply:
agree to the above and would like to add that you could utilize the colors in our mui default theme instead of assigning them with manual value.   as far as i remember, we dont have black text in our design but a dark 282828 color which is in our theme in the palette object, please look for it.

Reply:
done